.travis.yml

```diff
@@ -8,4 +8,6 @@ notifications:
   email: false
 before_install:
   - npm i -g npm@6.9.0
-install: npm ci
+  - npm install -g lerna
```

Comment:
so here, we will always run test for both repo when we do release or PR?

Reply:
@cakasuma I think so

Reply:
Yes, this will later change to test only affected sections of the app.